src/Microsoft.Azure.SignalR/HubHost/NegotiateHandler.cs

```diff
@@ -51,7 +51,10 @@ public async Task<NegotiationResponse> Process(HttpContext context, string hubNa
             var request = context.Request;
             var cultureName = context.Features.Get<IRequestCultureFeature>()?.RequestCulture.Culture.Name;
             var originalPath = GetOriginalPath(request.Path);
-            var provider = _endpointManager.GetEndpointProvider(_router.GetNegotiateEndpoint(context, _endpointManager.GetEndpoints(hubName)));
+
+            var endpoints = _endpointManager.GetEndpoints(hubName);
```

Comment:
revert the change.

The rule is to avoid including a file if only refactor changes are involved in this file. The refactor changes can be done in a separate PR.